certs/postgres/certs.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/mainflux/mainflux/certs"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+const duplicateErr = "unique_violation"
+
+var (
+	errSaveDB       = errors.New("failed to save certs to database")
+	errRetrieve     = errors.New("failed to retrieve certs  from database")
```

Comment:
You have 2 white space in this log. I would maybe use `failed to read certs from database`